XiEditor/CoreConnection.swift

```diff
@@ -66,23 +79,72 @@ class CoreConnection {
         } else {
             self.rpcLogWriter = nil
         }
-        let task = Process()
+        
+        let tmpErrLog = logDirectory.appendingPathComponent("xi_tmp.log").path
```

Comment:
Similar to above, this string appears in a few places. I would just add a `defaultCoreLogPath` var to the app delegate, and use that anywhere we need this path.